static/js/tools/choropleth/choropleth.tsx

```diff
@@ -615,12 +617,14 @@ class ChoroplethMap extends Component<PropsType, StateType> {
    *                 is country/USA.
    */
   private redirectToGeo = (geoDcid: string, curGeo: string): void => {
-    const url = new URL(window.location.href);
-    const currGeoId = url.searchParams.get("geoDcid");
-    let baseUrl = "/tools/choropleth";
-    baseUrl += buildChoroplethParams(["statVar", "pc", "mdom"]);
-    baseUrl += "&geoDcid=" + geoDcid;
-    baseUrl += "&bc=";
+    const currGeoId = new URLSearchParams(window.location.search).get(
+      "geoDcid"
+    );
+    let baseUrl = `/tools/choropleth${buildChoroplethParams([
+      "statVar",
+      "pc",
+      "mdom",
+    ])}&&geoDcid=${geoDcid}&bc=`;
```

Comment:
double ampersands? also looks like you could pass in geoDcid into the list of args for buildChoroplethParams to preserve.

Reply:
I will fix the double ampersands. buildChoroplethParams will set values to what is currently in the url, but we want the geoDcid for the next geo we're redirecting to

Reply:
Ah yes, thanks for catching that. Readability nit for future: the code should be localized with where it's used (i.e. get currGeoId right before using it).

I'll approve and let's submit.